test_runner/src/main/kotlin/ftl/gc/GcAndroidTestMatrix.kt

```diff
@@ -57,6 +57,7 @@ object GcAndroidTestMatrix {
             .setAdditionalApks(additionalApkGcsPaths.mapGcsPathsToApks())
             .setFilesToPush(otherFiles.mapToDeviceFiles())
             .setEnvironmentVariables(args, androidTestConfig)
+            .setDontAutograntPermissions(true)
```

Comment:
It should be depending on `grantPermissions` option: `all => false` and `none => true`

https://github.com/Flank/gcloud_cli/blob/master/google-cloud-sdk/lib/googlecloudsdk/api_lib/firebase/test/android/matrix_creator.py#L211

Reply:
ahh ok ok I understand. and done